packages/install-ses/install-ses.js

```diff
@@ -1,10 +1,15 @@
 /* global globalThis LOCKDOWN_OPTIONS process */
-// 'lockdown' appears on the global as a side-effect of importing 'ses'
-import 'ses';
```

Comment:
Please keep these lines, as the order of the imports matters (or at least *will* matter when #3891 lands).

I think it's fine to have both an `import` and `export` directive with the same `'ses'` module specifier.